UltimateTeam.Toolkit/Models/WatchlistResponse.cs

```diff
@@ -2,13 +2,13 @@
 
 namespace UltimateTeam.Toolkit.Models
 {
-    public class WatchlistResponse
+    public class WatchlistResponse : IListResponse
```

Comment:
Would be great if you could push another commit to your branch that removes ´virtual`. Just push to the same branch and they will show up here.

I can change it locally if you're not able to push another commit, your choice :smile: